WordPress/Classes/Models/Blog.h

```diff
@@ -63,6 +63,9 @@ typedef NS_ENUM(NSUInteger, BlogFeature) {
 @property (nonatomic, assign, readwrite) BOOL isMultiAuthor;
 @property (nonatomic, assign, readwrite) BOOL isHostedAtWPcom;
 @property (nonatomic, strong, readwrite) NSString *icon;
+@property (nonatomic, strong, readwrite) NSNumber *defaultCategory;
```

Comment:
I'd call this `defaultCategoryID`. Or maybe better, leave it as defaultCategory but make it a relationship that actually returns the `PostCategory`